diff --git a/node_modules/mini-css-extract-plugin/dist/index.js b/node_modules/mini-css-extract-plugin/dist/index.js
index 070dd27..70b3bc4 100644
--- a/node_modules/mini-css-extract-plugin/dist/index.js
+++ b/node_modules/mini-css-extract-plugin/dist/index.js
@@ -338,6 +338,9 @@ class MiniCssExtractPlugin {
           // no module found => there is a conflict
           // use list with fewest failed deps
           // and emit a warning
+          if (!bestMatch) {
+            break;
+          }
           const fallbackModule = bestMatch.pop();
           compilation.warnings.push(new Error(`chunk ${chunk.name || chunk.id} [mini-css-extract-plugin]\n` + 'Conflicting order between:\n' + ` * ${fallbackModule.readableIdentifier(requestShortener)}\n` + `${bestMatchDeps.map(m => ` * ${m.readableIdentifier(requestShortener)}`).join('\n')}`));
           usedModules.add(fallbackModule);
